pydocx/util/xml.py

```diff
@@ -40,6 +40,8 @@ def el_iter(el):
 def find_first(el, tag):
     """
     Find the first occurrence of a tag beneath the current element.
+
+    This method should not be used to navigate an OOXML schema
```

Comment:
We could do that... although technically since we're < 1.0 it's not required. In alpha, everything is pending deprecation.